integration/db_integration_test.go

```diff
@@ -55,106 +55,139 @@ import (
 	"github.com/gravitational/teleport/lib/utils"
 )
 
-// TestDatabaseAccessPostgresRootCluster tests a scenario where a user connects
-// to a Postgres database running in a root cluster.
-func TestDatabaseAccessPostgresRootCluster(t *testing.T) {
-	pack := setupDatabaseTest(t)
+// TestDatabaseAccess runs the database access integration test suite.
+//
+// It allows to make the entire cluster set up once, instead of per test,
+// which speeds things up significantly.
+func TestDatabaseAccess(t *testing.T) {
+	pack := setupDatabaseTest(t,
+		// set tighter rotation intervals
+		withLeafConfig(func(config *service.Config) {
+			config.PollingPeriod = 5 * time.Second
+			config.RotationConnectionInterval = 2 * time.Second
+		}),
+		withRootConfig(func(config *service.Config) {
+			config.PollingPeriod = 5 * time.Second
+			config.RotationConnectionInterval = 2 * time.Second
+		}),
+	)
+	pack.waitForLeaf(t)
+
+	t.Run("PostgresRootCluster", pack.testPostgresRootCluster)
```

Comment:
This is a bit risky: I can easily see someone missing adding a test to this list.

Reply:
It will mean that somebody wrote the `pack` suite test without running it.  
You can get the same behaviour when you wrongly name test like `TesttDatbaseFlow`

Reply:
Makes sense.